2.0/BoxTexturedNonPowerOfTwo/README.md

```diff
@@ -0,0 +1,13 @@
+# Box Textured
+
+This model uses a Non Power-Of-Two texture with REPEAT mode wrapping.  This is an edge case that is technically a valid gltf model, but needs some renderer work to resize the texture before uploading to the GPU.  According to the spec:
+
+Non-Power-Of-Two Texture Implementation Note: glTF does not guarantee that a texture's dimensions are a power-of-two. At runtime, if a texture's width or height is not a power-of-two, the texture needs to be resized so its dimensions are powers-of-two if the sampler the texture references
```

Comment:
The last statement is incomplete (compare with the spec). It may be better to put a link instead.